.github/workflows/main.yaml

```diff
@@ -39,6 +39,11 @@ on:
       - "Makefile"
       - "go.mod"
       - "go.sum"
+
+env:
+  KUBECTL_VERSION: 1.25.1
```

Comment:
Stick with same version as before for now, but we might want to upgrade to more recent version in a followup PR. Or I can add the renovate annotation to get automated updates on these version in a followup.